### PR TITLE
chore: update protos to use protoc-gen-gogofaster

### PR DESCRIPTION
**Pull request by NamanJain8**
With protoc-gen-gofast, we had some extra XXX_ fields. Using gogofaster removes these extra fields.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7336)

### CODE REVIEW

File: posting/size.go [hunk 1]
```diff
@@ -88,10 +88,7 @@ func calculatePostingListSize(list *pb.PostingList) uint64 {
 	var size uint64 = 1*8 + // Pack consists of 1 word.
 		3*8 + // Postings array consists of 3 words.
 		1*8 + // CommitTs consists of 1 word.
-		3*8 + // Splits array consists of 3 words.
-		0*8 + // XXX_NoUnkeyedLiteral consists of 0 words. because it is empty struct.
```
Comment (jarifibrahim) [inline posting/size.go:92]: Nice :+1:

**Review by jarifibrahim — COMMENTED** [review]
PR looks fine. @NamanJain8 can you verify if this isn't breaking anything?

**Review by manishrjain — APPROVED** [review]
:lgtm: Yup. Ensure it doesn't break anything.

> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7336#-:-MR_encp4penKssqPDIY:bx1q1z7)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @martinmr and @vvbalaji-dgraph)

**Comment by NamanJain8** [comment]
> PR looks fine. @NamanJain8 can you verify if this isn't breaking anything?

Verified. Backup/restore works fine across this change and `master`. Moreover, the same data directories work seamlessly across the change.

**Pull request merged**